batchflow/models/tf/base.py

```diff
@@ -1745,8 +1745,8 @@ def default_config(cls):
     def fill_params(cls, _name, **kwargs):
         """ Fill block params from default config and kwargs """
         config = cls.default_config()
-        _config = config.get(_name)
-        config = {**config['common'], **_config, **kwargs}
+        _config = Config(config.get(_name))
```

Comment:
`default_config` returns `Config` by design

Reply:
Firstly, `fill_params` is supposed to return a `dict` of kwargs which later is passed to functions. Make it an instance of `Config` might require some refactoring and testing.